script/Ciel/Day 14/14_01_CIEL14_1.txt

```diff
@@ -531,7 +531,7 @@ How often does the pain come? How   many times  per    second――per  minute
 }
 [sha:c082dfd9b9823b6d3670c48016083d0ed4ce8f53]{
 -- Page 29, Offset 37100.
--- いたい。どうにかしようと、ナイフで掌をザクザクと刺してみた。ダメだ。ぜんぜんイタク、ない。そうか。たぶんきっと線の  セイだ。眼鏡。眼鏡をかければ、いつものように、こんな異常は、収まって
+-- いたい。どうにかしようと、ナイフで掌をザクザクと刺してみた。ダメだ。ぜんぜんイタク、ない。そうか。たぶんきっと線の　セイだ。眼鏡。眼鏡をかければ、いつものように、こんな異常は、収まって
 It hurts.
  Just to do something, %{i}anything%{/i}, I stab my palm over and over again with my knife.
  It doesn't  work.
```

Comment:
spooky change

Reply:
result of me bulk replacing some unicode spaces that got in here from a suggestion earlier